base/CMakeLists.txt

```diff
@@ -51,6 +52,33 @@ therock_cmake_subproject_provide_package(rocm-core rocm-core lib/cmake/rocm-core
 therock_cmake_subproject_activate(rocm-core)
 
 
+################################################################################
+# amdsmi
+################################################################################
+
+if(NOT WIN32)  # TODO(#36): Enable on Windows and/or make subproject inclusion generally optional
```

Comment:
Can you add a row to the table at https://github.com/ROCm/TheRock/blob/main/docs/development/windows_support.md#supported-subprojects for this?

Reply:
Sure! Pushing a commit asap.